calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -1,62 +1,97 @@
 ---
-description: Identify vulnerabilities in Kubernetes clusters.
+description: Detect vulnerabilities in a Kubernetes cluster.
 ---
 
-# Enable automatic image scanning inside a cluster
+# Scan all images in a Kubernetes cluster
 
-## Big picture
+Scan all images in a Kubernetes cluster for continuous clean bill of health and defense in depth.
 
-Scan Kubernetes workloads automatically using {{prodname}} Image Assurance cluster scanner.
+Common use cases for scanning in a Kubernetes cluster are: 
 
+- Images may pass scanning during the build phase, but they could contain vulnerabilities days or weeks later
+- Third-party images pulled from public registries are often not scanned in build pipelines and can contain Critical or High vulnerabilities
+- Application teams often build one-off images outside their pipeline to make an emergency patch and fix a critical bug
 
-## Before you begin
+## Image Assurance cluster scanner
 
-**Cluster requirements**
+The cluster scanner is out-of-the-box ready to enable without configuration. Running images are checked for new vulnerabilities every 24 hours, and scan results are reported back to {{prodname}} Manager UI in the Image dashboard.
```

Comment:
Not sure we need to clarify, but it feels like this sentence implies that the cluster scanner is the one that does 24 hr rescanning. 

That's not actually true. 

The cluster scanner does the first step of a vulnerability scan: scanning images for the software bill of materials (SBOM). The SBOM is then uploaded to the cloud and in the cloud, another component does step two: match packages in the SBOM with known CVEs. 

We probably don't want to mention this level of detail. But we also should try to avoid giving the user the wrong impression of what the cluster scanner itself does.

Reply:
I agree that it should be technically correct. Rewrote.